docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
+
+1. Open the Teleport configuration file—by default, `/etc/teleport.yaml`—in an editor on the computer 
+where you installed the Teleport agent.
+
+1. Locate the `labels` configuration under the `ssh_service` section.
+
+1. Add the static label key and value. 
+   For example, add `environment` as the label key and `dev` as the value:
+
+   ```yaml
+   ssh_service:
+     enabled: true
+     labels:
+       environment: dev
+   ```
+
+1. Save your changes and close the file.
+
+1. Start Teleport with the invitation token you saved in the `INVITE_TOKEN` environment variable:
+   
+   ```code
+   $ sudo teleport start --token=${INVITE_TOKEN?}
+   ```
+
+1. Verify that you have added the label by running the following command 
+on your local computer. 
+      
+   ```code
+   $ tsh ls --query 'labels["environment"]=="dev"'
+   ```
+
+   You should see output similar to the following:
+
+   ```code
+   Node Name        Address    Labels                                                                                                       
+   ---------------- ---------- ------------------------------------------------------------------------
+   ip-192-168-13-57 ⟵ Tunnel   environment=dev,hostname=ip-192-168-13-57,teleport.internal/resource-id
+   ```
+
+   #### Checking the status of your server
+
+   If you don't see your server listed when you query for the label you added, you should verify that the 
+   SSH Service is running on the server. Check the log for the server to verify that there are 
+   messages similar to the following:
+   
+   ```text
+   2023-08-07T22:22:21Z INFO [NODE:1]    Service is starting in tunnel mode. pid:149932.1 service/service.go:2630
+   2023-08-07T22:22:21Z INFO [UPLOAD:1]  starting upload completer service pid:149932.1 service/service.go:2723
+   2023-08-07T22:22:21Z INFO [PROC:1]    The new service has started successfully. Starting syncing rotation status...
+   ```
+   
+   #### Checking your user profile
+   
+   If the SSH Service is running on the server, verify that your current Teleport user has a login on the local host.
+   You can check the status of your user account by running the following command:
+   
+   ```code
+   $ tsh status
+   ```
+      
+   You should see output similar to the following with at least one login listed for your current user:
+   
+   ```code
+   > Profile URL:        https://ajuba-aws.teledocs.click:443
+     Logged in as:       teleport-admin
+     Cluster:            teleport-aws.example.com
+     Roles:              access, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         enabled
+     Valid until:        2023-08-08 10:08:46 +0000 UTC [valid for 10h36m0s]
+     Extensions:         login-ip, permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+   ```
+   
+   If no valid local logins have been assigned to the user, you should update your current user profile to include
+   at least one valid local login.
```

Comment:
```suggestion
   at least one valid login.
```

Not sure what you mean by local, as these are usernames used to log in to a remote system. I think the sentence reads just as well without it.